src/cli/serve/read_yaml_config.js

```diff
@@ -38,38 +43,40 @@ const deprecatedSettings = {
   'server.xsrf.token': 'server.xsrf.token is deprecated. It is no longer used when providing xsrf protection.'
 };
 
-module.exports = function (path) {
-  if (!path) return {};
-
-  let file = yaml.safeLoad(fs.readFileSync(path, 'utf8'));
-
-  function apply(config, val, key) {
-    if (_.isPlainObject(val)) {
-      _.forOwn(val, function (subVal, subKey) {
-        apply(config, subVal, key + '.' + subKey);
-      });
-    }
-    else if (_.isArray(val)) {
-      config[key] = [];
-      val.forEach((subVal, i) => {
-        apply(config, subVal, key + '.' + i);
-      });
-    }
-    else {
-      _.set(config, key, val);
-    }
-  }
+export default function (paths) {
+  if (!paths) return {};
 
-  _.each(deprecatedSettings, function (message, setting) {
-    if (_.has(file, setting)) console.error(message);
+  const files = [].concat(paths).map(path => {
```

Comment:
Actually I'm doing it to support receiving both an array of paths or a single path.